index.js

```diff
@@ -49,8 +55,13 @@ Do the following:
    HINT: look up the Number method
 */
 
+let year = '1999'
 
+console.log(year)
+console.log(Number(year))
```

Comment:
Another way of turning the string into an integer is using the parseInt() function